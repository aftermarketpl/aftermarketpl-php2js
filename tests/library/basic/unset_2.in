$a = [];
$a[1] = "Jeden";
unset($a[2]);
return isset($a[1]);
