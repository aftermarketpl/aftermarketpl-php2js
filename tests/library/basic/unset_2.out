var a;
a = [];
a[1] = "Jeden";
delete a[2];
return (typeof a[1] !== 'undefined');
